examples/http-1-1-to-2-0-protocol-switch/http_1_1_to_2_0_protocol_switch.bal

```diff
@@ -1,16 +1,15 @@
 import ballerina/http;
 
 // HTTP version is set to 2.0.
-http:Client http2serviceClientEP =
-        check new ("http://localhost:7090", {httpVersion: "2.0"});
+http:Client http2serviceClientEP = check new ("http://localhost:7090", {httpVersion: "2.0"});
```

Comment:
```suggestion
http:Client http2serviceClientEP = check new ("http://localhost:7090");
```

Reply:
Not needed as defaults to 2.0

Reply:
Fixed with [18d6eb246533e444b42727ad2010b376160c5cb1](https://github.com/ballerina-platform/ballerina-distribution/pull/3024/commits/18d6eb246533e444b42727ad2010b376160c5cb1)